__tests__/component-options.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-env jest */
+const configPaths = require('../config/paths.json')
+const PORT = configPaths.port
+
+let browser
+let page
+let baseUrl = 'http://localhost:' + PORT
+
+beforeAll(async (done) => {
+  browser = global.browser
+  page = await browser.newPage()
+  done()
+})
+
+afterAll(async (done) => {
+  await page.close()
+  done()
+})
+
+describe('Component page', () => {
+  it('should contain a "Nunjucks" tab heading', async () => {
+    await page.goto(baseUrl + '/components/back-link/', { waitUntil: 'load' })
+
+    const nunjucksTabHeadings = await page.evaluate(() => Array.from(document.querySelectorAll('.js-tabs__item a')).filter(element => element.textContent === 'Nunjucks'))
```

Comment:
that's legit, alternatively you could use the spread operator 
`[...document.querySelectorAll('.js-tabs__item a')]